### ISSUE TITLE
Gulp task never ends

### ISSUE THREAD

**Issue by syko** [sindresorhus/gulp-imagemin#20]
Hi,

Not sure what the problem is here but when using gulp-imagemin, the gulp task never ends and leaves the process hanging (manual ctrl+c required).

For example:

```
gulp.task('build-images', function() {
  return gulp.src(paths.images)
    .pipe(imagemin({pngquant:true}))
    .pipe(gulp.dest(paths.build + "images/"));
});
```

Runnig `gulp build-images` reports:

```
[gulp] Running 'build-images'...
[gulp] gulp-imagemin: ✔ image1.png (saved 176.83 kB)
[gulp] gulp-imagemin: ✔ image2.png (saved 2.45 kB)
[gulp] Finished 'build-images' in 190 ms
```

Gulp reports as if the task has ended, but the process never ends.
Using gulp-imagemin v0.2.0 and gulp 3.5.6

**Comment by moimael** [comment]
I still have this issue with gulp-imagemin 2.2.1

**Comment by frederico** [comment]
I still have this issue with  gulp-imagemin 2.4.0

**Comment by elambro** [comment]
I have this issue with gulp 3.91 and imagemin 3.3.0

**Comment by gRoussac** [comment]
Hi,

Apparently also on 3.3.0. Cheers

**Comment by xahon** [comment]
the same issue with "gulp-imagemin": "^4.1.0"

**Comment by santimirandarp** [comment]
same ^7.1.0

**Comment by Andreas-Schoenefeldt** [comment]
I had the same - solved by using gulp 5.0.0
